hyperjumptech/monika
# Add Google chat notification channel

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Closes issue #494 
2. Adds google chat as a notification channel

## How did you implement / how did you fix it  
1.  Add handling for type 'google-chat'
2. Add  json formatting specific for google chat.

## How to test  
1. Create a new notification of type 'google-chat'
2. Add the spaces/google chat bot and generate the url webhook like in the google instruction
3. Run monika and observe notifications on Google Chat space

startup message
![image](https://user-images.githubusercontent.com/964106/140691722-fcb98810-eee9-4d14-baa6-51834772bc15.png)

incident message
![image](https://user-images.githubusercontent.com/964106/140691862-fb79022d-a6ba-479b-b3a1-a28c3c7d2f4f.png)
